docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-kubernetes.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrade YugabyteDB Anywhere installation on Kubernetes
+headerTitle: Upgrade YugabyteDB Anywhere installation on Kubernetes
+linkTitle: Upgrade Kubernetes installation
+description: Upgrade YugabyteDB Anywhere installation on Kubernetes
+menu:
+  preview:
+    identifier: upgrade-yp-kubernetes
+    parent: upgrade
+    weight: 80
+isTocNested: true
+showAsideToc: true
+---
+
+You can use [Helm](https://helm.sh/) to upgrade your YugabyteDB Anywhere installed on [Kubernetes](https://kubernetes.io/) to a newer version.
```

Comment:
It would be nice to also ask users to do a `helm repo update` first so that they can pick up newer versions from the charts. 

The command would just be `helm repo update` and it is analogous to the `helm repo add` command in the install step.

Reply:
cc @lizayugabyte

Reply:
@iSignal Sure, I'll add this info via a new PR